src/main/kotlin/pl/newbies/event/application/model/EventRequest.kt

```diff
@@ -40,8 +51,13 @@ data class AddressRequest(
 
     init {
         validate(this) {
-            validate(AddressRequest::city).isNotBlank()
-            validate(AddressRequest::place).isNotBlank()
+            validate(AddressRequest::city)
+                .isNotBlank()
+                .hasSize(max = 50)
+                .maxLines(5)
```

Comment:
i don't know any 5 line city

Reply:
bruh, I've changed wrong field. That's what you get if you do something in a hurry :/